src/main/resources/js/lgtm-addon.js

```diff
@@ -89,30 +87,38 @@ function handleProjectChange(event) {
 		loadPriorities(AJS.$(projectFieldId).select2('val'));
 	}
 }
-
+	
 function loadIssueType(projectKey) {
 	$.ajax({
 		url : AJS.contextPath() + "/rest/api/2/project/" + projectKey + "/statuses"
 	}).done(
 	function(issueTypes) {
 		var lgtmIssueType = issueTypes.find(function (element) {
-			return element.name === "LGTM Alert";
+			return element.name.toLowerCase() === "LGTM alert".toLowerCase();
 		});
 		
 		if (lgtmIssueType === undefined) {
+			var children = AJS.$("#message-context").children();
+			
+			for (var i = 0; i < children.length; i++) {
```

Comment:
I think this clean-up is best executed regardless of whether `lgtmIssueType` is undefined, so it'll clear it if someone comes back to the page and re-selects.